### DISCUSSION TITLE
Voiceless videos are converted to gifs when shared

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#17007] [General]
Why does Mastodon convert voiceless videos to gifs when shared, and is there a way to stop this behavior?

**Comment by Gargron** [comment]
It does not convert them to the GIF file format, it displays them as "GIFs", which in reality are soundless videos. Both "GIFs" and videos are mp4 and rendered using a `<video>` element, the only difference is whether video player controls are displayed and whether it autoplays and loops. This is to allow uploading of "GIFs" as mp4/webm.

**Reply by ghost** [reply]
> It does not convert them to the GIF file format, it displays them as "GIFs", which in reality are soundless videos. Both "GIFs" and videos are mp4 and rendered using a `<video>` element, the only difference is whether video player controls are displayed and whether it autoplays and loops. This is to allow uploading of "GIFs" as mp4/webm.

So, there is no way to display video player controls on these videos? Is there an open issue about that, if not I am thinking of opening one; I want the video control features on them.
